bitnami/fluent-bit/values.yaml

```diff
@@ -0,0 +1,681 @@
+## @param tolerations Tolerations for pod assignment
+## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
+##
+tolerations: []
+## @param topologySpreadConstraints Topology Spread Constraints for pod assignment
+## https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/
+## The value is evaluated as a template
+##
+topologySpreadConstraints: []
+## @param schedulerName Alternative scheduler
+## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+##
+schedulerName: ""
+## @param updateStrategy.type Fluent Bit deployment strategy type
+## @param updateStrategy.rollingUpdate Fluent Bit deployment rolling update configuration parameters
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/#strategy
+##
+updateStrategy:
+  type: RollingUpdate
+  rollingUpdate: {}
+## @param extraVolumes Optionally specify extra list of additional volumes for fluent-bit container
+##
+extraVolumes: []
+## @param initContainers Add additional init containers to the fluent-bit pods
+##
+initContainers: []
+## @param sidecars Add additional sidecar containers to the fluent-bit pods
+##
+sidecars: []
+
+## @section Fluent Bit configuration
+##
+config:
+  ## @param config.flush Interval to flush output (seconds)
+  ##
+  flush: 1
+  ## @param config.logLevel Diagnostic level (error/warning/info/debug/trace)
+  ##
+  logLevel: info
+  ## @param config.service [string] Defines the global behaviour of the Fluent Bit engine.
+  ##
+  service: |
+    [SERVICE]
+        Flush        {{ .Values.config.flush }}
+        Daemon       Off
+        LogLevel     {{ .Values.config.logLevel }}
+        Config_Watch On
+        HTTP_Server  On
+        HTTP_Listen  0.0.0.0
+        HTTP_Port {{ .Values.containerPorts.http }}
+  ## @param config.inputs [string] Defines the source from where Fluent Bit can collect data
+  ## https://docs.fluentbit.io/manual/pipeline/inputs
+  ##
+  inputs: |
+    [INPUT]
+        Name cpu
+
+    [INPUT]
+        Name systemd
+        Tag host.*
+        Systemd_Filter _SYSTEMD_UNIT=kubelet.service
+        Read_From_Tail On
+
+  ## @param config.filters [string] Set of plugins that can be used to filter, modify, or enrich log data that is processed by Fluent Bit.
+  ## https://docs.fluentbit.io/manual/pipeline/filters
+  ##
+  filters: |
+    [FILTER]
+        Name                kubernetes
+        Match               kube.*
+        Kube_URL            https://kubernetes.default.svc:443
+        Kube_CA_File        /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+        Kube_Token_File     /var/run/secrets/kubernetes.io/serviceaccount/token
+        Kube_Tag_Prefix     kube.var.log.containers.
+        Merge_Log           Off
+        Merge_Log_Key       log_processed
+        K8S-Logging.Parser  On
+        K8S-Logging.Exclude Off
+  ## @param config.outputs [string] Outputs to send the collected data to different destinations
```

Comment:
This filter is only being applied in the daemonset version, as the deployment one is not mounting that folder. Is this expected? Which is the default configuration that is consuming when using deployment (which is set as default)